Tests/FluentAssertions.Specs/Equivalency/CollectionEquivalencySpecs.cs

```diff
@@ -1301,21 +1301,6 @@ public void When_asserting_equivalence_of_collections_and_configured_to_use_runt
             act.Should().Throw<XunitException>("the types have different properties");
         }
 
-        [Fact]
-        public void When_injecting_a_null_config_to_non_generic_overload_it_should_throw()
-        {
-            // Arrange
-            ICollection collection1 = null;
-            ICollection collection2 = null;
-
-            // Act
-            Action act = () => collection1.Should().BeEquivalentTo(collection2, config: null);
```

Comment:
This used to call `BeEquivalent(IEnumerable, Func<...>)`, but that overload is now removed and it resolves to `BeEquivalent(object, Func<...>)` which we have other tests for.